src/app/static/src/app/store/modelCalibrate/modelCalibrate.ts

```diff
@@ -18,8 +18,7 @@ export interface ModelCalibrateState extends ReadyState, WarningsState {
     complete: boolean
     generatingCalibrationPlot: boolean
     calibratePlotResult: any,
```

Comment:
This can be specific to the response type now in the generated file.

Reply:
Do you mean the calibratePlotResult?

Reply:
yes!